auction-deploy/auction_deploy/cli.py

```diff
@@ -154,6 +152,45 @@ def deploy(
     click.echo("Validator slasher address: " + contracts.slasher.address)
 
 
+@main.command(short_help="Start the auction at corresponding address.")
+@click.option(
+    "--auction-address",
+    help='The address of the auction contract to be started, "0x" prefixed string',
+    type=str,
```

Comment:
We could validate if it is an Address with a callback, see
https://github.com/trustlines-protocol/tlbc-monitor/blob/master/monitor/main.py#L358

Reply:
you can hint is is an Address with 
https://github.com/trustlines-protocol/contracts/blob/master/py-deploy/tldeploy/cli.py#L80